packages/nextlove/src/generators/lib/parse-routes-in-package.ts

```diff
@@ -37,6 +37,14 @@ export const parseRoutesInPackage = async (opts: {
       const { default: routeFn } = await require(path.resolve(p))
 
       if (routeFn) {
+        if (routeFn._routeSpec?.excludeFromOpenApi) {
+          console.warn(
+            chalk.yellow(
+              `Ignoring "${p} because it was excluded from openAPI generation"`
+            )
+          )
+          return
+        }
```

Comment:
```suggestion
        if (routeFn._routeSpec?.excludeFromOpenApi) {
          console.log(
            chalk.gray(
              `Ignoring "${p} because it was excluded from openAPI generation"`
            )
          )
          return
        }
```